src/domains/global/reducer.ts

```diff
@@ -79,6 +84,7 @@ export type StateT = {
   }
   hoveredX: number | null
   hasWindowFocus: boolean
+  globalPause: boolean
```

Comment:
I'm afraid that is a state duplication, but correct me if i'm wrong. Until now, the "pause" mode could be calculated based on globalPanAndZoom, hoveredX, etc. It could be done as a separate selector (something like computed property based on already existing state).
Now if you add one more property it suddenly becomes more complicated because you can have unstable/impossible states, you need to always keep both in sync.

@allelos What do you think?

Reply:
It would be state duplication if `pause` was only dependent on `globalPanAndZoom`, `hoveredX`, etc. The `pause` state in that case would be a derived property. 
From now on pausing will not only react to the above dependencies but also to external dependencies such as of manually pausing. It describes a different state.
Also keep in mind that during a forced pause we would also like to have access to `globalPanAndZoom` and `hoveredX` properties for showing values when hovering, panning etc without messing with the paused state.
In my opinion, this would be a lot more complex without having a global play / pause state. Also, complexities of 1 to 1 relationships will be, most of the time, a lot simpler to use and reason about than derived ones etc.

Reply:
@allelos can you briefly explain when will be the situation where `globalPause` will be `true`, but `globalPanAndZoom` and `hoveredX` will be `null`?

Reply:
previously to manually pause, `setGlobalPanAndZoom` action was send.
My problem is that we should not change all occurences where `globalPanAndZoom` was used to manipulate play/pause, and switch to `globalPause`, do you agree?

Re: complexities, IMHO derived properties (vs state duplication) are just less buggy, because you need to handle less nr of different scenarios.

In this scenario, i would combine `globalPanAndZoom` with `globalPause`, for example by adding `wasManuallyTriggered` flag inside `globalPanAndZoom` object. But of course your way will work too, just we need to update always both properties - `globalPanAndZoom` and `globalPause`.

Reply:
> @allelos can you briefly explain when will be the situation where `globalPause` will be `true`, but `globalPanAndZoom` and `hoveredX` will be `null`?

The simplest case would be when just clicking the play / pause button. 

> My problem is that we should now change all occurences where `globalPanAndZoom` was used to manipulate play/pause, and switch to `globalPause`, do you agree?

Yes and no. It's true that `globalPanAndZoom` and `globalPause` are closely related especially compared to `globalPause` and `hoveredX`. But they are not describing the same thing. `globalPause` is a superset of `globalPanAndZoom`. Pause happens because of pan and zoom, but  pan and zoom doesn't happen because of pausing. 

If we were to combine them I would go with `globalPause` containing pan and zoom instead of the opposite. We can do this in the future, but I don't see any problem (besides updating both) with having them decoupled , at least for now.

@jacekkolasa wdyt?

Reply:
> The simplest case would be when just clicking the play / pause button.
I had situations like this (for example when people were clicking "pause" button below chart), in those cases setGlobalPanAndZoom was dispatched. So i was using different state for exactly the same action, that's why i was proposing that we unify this. Or maybe we don't need to since new refactor is coming anyway :)